README.md

```diff
@@ -179,6 +179,15 @@ python -m pytest 'tests/' -v
 
 Ensure you have your environment configured correctly (as described above).
 
+### Unit Test Warnings
+
+There are a few warnings shown by pypi, based on old package versions. These can be resolved by running the following commands:
```

Comment:
Is there a corresponding requirements.txt we should update for this?

Reply:
Good point, updated! The protobuf library is a dependency, so doesn't need to be specified directly in the requirements.txt

Reply:
Updated the README as well